src/screens/CreatePost/index.js

```diff
@@ -448,7 +447,7 @@ const CreatePost = () => {
   const checkTaggingUser = () => {
     const mapTagUser = taggingUsers.map((data) => {
       const findData = allTaggingUser.find((dataUser) => dataUser.username === data);
-      return findData.user_id;
+      return findData?.user_id;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on adding optional chaining to handle the case where `findData` is undefined. This will prevent potential runtime errors when trying to access the `user_id` property of `undefined`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->